sql/driver/conn.go

```diff
@@ -186,6 +188,10 @@ func (c *conn) Send(call sqlwire.Call) (*rows, error) {
 			} else if datum.StringVal != nil {
 				t[j] = *datum.StringVal
 			}
+			if t[j] != nil && !driver.IsValue(t[j]) {
```

Comment:
You want me to convert the string to a []byte and use IsScanValue? Why? Is it because you are converting []byte to string in SQL SELECT.